ignite/services/network/network.go

```diff
@@ -101,13 +110,13 @@ func CollectEvents(ev events.Bus) Option {
 // New creates a Builder.
 func New(cosmos CosmosClient, account cosmosaccount.Account, options ...Option) Network {
 	n := Network{
-		cosmos:        cosmos,
-		account:       account,
-		campaignQuery: campaigntypes.NewQueryClient(cosmos.Context()),
-		launchQuery:   launchtypes.NewQueryClient(cosmos.Context()),
-		profileQuery:  profiletypes.NewQueryClient(cosmos.Context()),
-		rewardQuery:   rewardtypes.NewQueryClient(cosmos.Context()),
-		stakingQuery:  stakingtypes.NewQueryClient(cosmos.Context()),
+		Node:                    NewNode(cosmos),
```

Comment:
```suggestion
```

Is there a good reason adding the Node public field here? Both Network and Node has RewardsInfo methods and embedding this makes the usage of that func ambiguous. 

Is there a way/or does it makes sense if we can merge these RewardsInfo funcs into one function but make configurable through `Options`s so it can be both used for SPN and a chain?

Reply:
No particular reason, it's only because I used the `Node` as an inheritance, but I moved now for a private field: https://github.com/ignite-hq/cli/pull/2516/commits/b6008938d06a8cf5b4e168bcd63e6c1d333f3231

It will be hard to merge the `RewardsInfo` from the network and node. They have different returns and also make the queries in different block heights.